From 07298646d823d23137664f9ab320154af7cab78c Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Thu, 15 Apr 2021 10:59:13 +0300
Subject: [PATCH] [SW-0] gaudi2_arc_gnu: fix compilation error

Below commit needs ScratchPad SRAM to pass a
fit that contains nic related info. This patch
caused a compile error in gaudi2_arc_gnu for
a missed definition of SPSRAM size.

    [SW-36011] gaudi: add spsram cmd for passing fit

    Add SPSRAM command so that a fit can be passed
    from fw-loader using Scratch pad SRAM.

This patch fixes the compilation error.

Change-Id: Ie866d47602755834d630d028ee52efb4750b379c
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arc_gnu/common.h b/gaudi2_arc_gnu/common.h
index 9a9712e..f2cb811 100644
--- a/gaudi2_arc_gnu/common.h
+++ b/gaudi2_arc_gnu/common.h
@@ -18,6 +18,7 @@
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
 
 #define HL_SP_SRAM_BASE		0x1000007FF7FE0000ULL
+#define HL_SP_SRAM_SIZE		0x10000
 #define PCIE_FW_SRAM_BASE	0x27FF0000
 #define FW_SRAM_BASE		0x27FD0000
 #define SP_SRAM_BASE		0x27FE0000
